data/schema.yaml

```diff
@@ -93,8 +93,14 @@ components:
             articleType:
               enum:
                 - analysis
+                - askTheAudience
```

Comment:
What is the context for adding these enums? Can they be added in the separate PR? We should keep adding to our schema distinct from other pieces of work.

Reply:
These types are part of the current scenarios but were never added to the schema, without them the validation fails and therefore the test suite can't run and Travis complains and I would have to `--no-verify` to push any changes. 
It is likely they will be updated within https://github.com/bbc/simorgh/issues/380 which will be merged before this PR.

Reply:
Okay. Please remove these changes and add this PR as blocked from merging dependent on https://github.com/bbc/simorgh/pull/380 being merged.